### ISSUE TITLE
Create ORKL connector

### ISSUE THREAD

**Issue by nino-filigran** [OpenCTI-Platform/connectors#1672]
## Use case

Create a connector for: https://orkl.eu/search?orkl_library%5BsortBy%5D=orkl_library%3Ats_creation_date%3Adesc
## Current Workaround

None
## Proposed Solution

Create connector 

## If the feature request is approved, would you be willing to submit a PR?

Yes / No (Help can be provided if you need assistance submitting a PR)

**Comment by sameershaik-coder** [comment]
Hello Team, I would like to work on developing this connector for orkl. Based on my research so far, this api provides a list of reports. Each report also has information related to threat actors. So we need to create a report for each library entry and associated threat actors and create a relationship between them. 

Is this understanding correct? Please let me know.

Below is the response from orkl api for your reference: 
{
  "data": [
    {
      "id": "a0fa940b-9de4-44bc-b7ac-c11f989129a8",
      "created_at": "2024-02-03T02:07:00.228313Z",
      "updated_at": "2024-02-06T02:18:13.919162Z",
      "deleted_at": null,
      "sha1_hash": "afa62ace20b3489a9b137a3b8091813c9d96373d",
      "title": "PowerPoint Presentation",
      "authors": "",
      "file_creation_date": "2024-01-22T11:51:43Z",
      "file_modification_date": "2024-01-22T11:51:43Z",
      "file_size": 3548713,
      "plain_text": "NSPX30\nA sophisticated AitM-enabled implant\nevolving since 2005\n\nFacundo Munoz\nMalware Researcher\n\nFacundo Munoz\n• Malware Researcher at ESET since 2021\n• Hunting and analysing APT malware\n\nfacundo.munoz@eset.com\n\nAgenda\nDiscovery of the NSPX30 implant\nOur research, evolution of NSPX30 and attribution to Blackwood APT\nCase study of an initial access via AitM\nThe NSPX30 implant and its features\nThe China-aligned AitM club\nConclusion\n\nIn the beginning there were… many detections\n\nThreat\nmagnet\ncomx3.dll\n\n• Evasive Panda\n• LuoYu\n• LittleBear\n\nmsnsp.dll\n\ncomx3.dll\n\nNSPX30 implant components\n\nThe NSPX30 implant\ndrops\n\nexecutes\n\nloads\n\nRsStub.exe\n\nDropper\n\nloads\n\ncomx3.dll\n\ncomx3.dll.txt\n\nindex.dat\n\nestablishes\npersistence\n\nEmbedded plugins\ndrops\n\nloads\n\nexecutes\n\nLegitimate\nprocess\n\nloads\n\nmsnsp.dll\n\ndownload\nand load\n\nloads\n\nmshlp.dll\n\nWIN.cfg\n\nmsfmtkl.dat\ndownloads\nloads\n\nPlugins\n\nMalware paleontology\n\nTimeline of NSPX30 and its ancestors\n\nFirst and last known sample of NSPX30\n\nFirst and last known sample of DCM (aka Dark Specter)\n\nFirst and last known sample of Project Wood\n\n2005\n\n2008\n\nNSPX30 detected in ESET\ntelemetry\n\n2013 2014\n\n2018 2019 2020\n\n2022 2023 2024\n\nCompilation timestamps, to trust or not to trust\n\nProject Wood backdoor\nMainFuncOften.dll\n\nCompiled on\n2005-01-09 08:21:22\n\nProject Wood dropper\nUnknown file name \n\nCompiled on\n2005-01-09 08:21:39\n\nCompilation timestamps, to trust or not to trust\n\nProject Wood backdoor\nMainFuncOften.dll\n\nCompiled on\n2005-01-09 08:21:22\n17 seconds difference!\n\nProject Wood dropper\nUnknown file name \n\nCompiled on\n2005-01-09 08:21:39\n\nUPX version\nDropper\n\n000003D0\n000003E0\n000003F0\n00000400\n00000410\n00000420\n00000430\n\n00 00 00 00 00 00 00 00 00 00 00 31 2E 32 34 00\n55 50 58 21 0C 09 05 09 C6 BF B8 96 B6 DB 30 81\nE7 39 04 00 0D 48 01 00 00 20 04 00 26 0A 00 29\n7F FF FF FF 51 53 55 56 8B 35 1C C0 40 00 8D 44\n24 0C 57 50 68 06 00 02 00 BD B4 E0 1F 6A 00 55\n68 ED B7 ED 7F 15 00 80 FF D6 8B 3D 20 3E 68 AC\n28 FF 74 24 14 FF D7 0A 10 91 B7 B5 ED 8B 1D 24\n\nUPX 1.24 was\nreleased in 2003\n\n...........1.24.\nUPX!....¿¸–¶Û0..\nç9...H......&..)\n.ÿÿÿQSUV‹5.À@..D\n$.WPh....½´à.j.U\nhí·í...€ÿÖ‹=.>h¬\n(ÿt$.ÿ×..‘·µí‹.$\n\nRich header metadata\n\nVisual Studio 6.0 was released in 1998\nAssessment: high confidence that is unlikely that attackers modified all these indicators.\n\nProject Wood aka PeerWoodCOften\nDropper\n\n00000000\n00000010\n00000020\n00000030\n00000040\n00000050\n00000060\n\n00 00 00 00 00 00 00 00 00 00 00 00 00 00 0E 00\n50 00 65 00 65 00 72 00 57 00 6F 00 6F 00 64 00\n43 00 4F 00 66 00 74 00 65 00 6E 00 00 00 00 00\n0C 00 48 00 65 00 6C 00 6C 00 6F 00 20 00 57 00\n6F 00 72 00 6C 00 64 00 21 00 00 00 00 00 0E 00\n50 00 45 00 45 00 52 00 57 00 4F 00 4F 00 44 00\n43 00 4F 00 46 00 54 00 45 00 4E 00 00 00 00 00\n\n................\nP.e.e.r.W.o.o.d.\nC.O.f.t.e.n.....\n..H.e.l.l.o...W.\no.r.l.d.!.......\nP.E.E.R.W.O.O.D.\nC.O.F.T.E.N.....\n\nPeerYou RAT\n\nOpen sourced in 2001\n\nPossibly of Chinese origin\n\nStill around in Chinese websites\n\nMany variants\n\nTimeline of publications about NSPX30’s ancestors\n\nTimeline of publications about NSPX30’s ancestors\nFirst and last known sample of NSPX30\nFirst and last known sample of DCM\n(aka Dark Specter)\nFirst and last known sample of Project Wood\n\nNSPX30 detected in ESET\ntelemetry\n\n2011\n2005\n\n2008\n\n2016\n2013\n\nFirst public report on\nProject Wood by\nFrankie Li\n\n2018 2019 2020\n\n2022 2023 2024\n\nTimeline of publications about NSPX30’s ancestors\nFirst and last known sample of NSPX30\nFirst and last known sample of DCM\n(aka Dark Specter)\nFirst and last known sample of Project Wood\n\nNSPX30 detected in ESET\ntelemetry\n\n2014\n\n2011 2012\n2005\n\n2008\n\n2013\n\nFirst public report on\nProject Wood by\nFrankie Li\nFirst public\nreport on DCM\nby Jianming\n\n2016\n2018 2019 2020\n\n2022 2023 2024\n\nTimeline of publications about NSPX30’s ancestors\nFirst and last known sample of NSPX30\nFirst and last known sample of DCM\n(aka Dark Specter)\nFirst and last known sample of Project Wood\n\nNSPX30 detected in ESET\ntelemetry\n\n2014\n\n2011 2012\n2005\n\n2008\n\n2013\nG DATA publishes\nreport about\nOperation TooHash\nFirst public report on\nProject Wood by\nFrankie Li\nFirst public\nreport on DCM\nby Jianming\n\n2016\n2018 2019 2020\n\n2022 2023 2024\n\nTimeline of publications about NSPX30’s ancestors\nFirst and last known sample of NSPX30\nFirst and last known sample of DCM\n(aka Dark Specter)\nFirst and last known sample of Project Wood\n\nNSPX30 detected in ESET\ntelemetry\n\n2014\n\n2011 2012\n2005\n\n2008\n\n2016\n2018 2019 2020\n\n2013\nG DATA publishes\nreport about\nOperation TooHash\nFirst public report on\nProject Wood by\nFrankie Li\nFirst public\nreport on DCM\nby Jianming\n\nTencent publishes\na report on\nDCM/Dark Specter\n\n2022 2023 2024\n\nTimeline of publications about NSPX30’s ancestors\nFirst and last known sample of NSPX30\nFirst and last known sample of DCM\n(aka Dark Specter)\nFirst and last known sample of Project Wood\n\nNSPX30 detected in ESET\ntelemetry\n\n2014\n\n2011 2012\n2005\n\n2008\n\n2016\n2018 2019 2020\n\n2013\n\n2022 2023 2024\n\nG DATA publishes\nreport about\nOperation TooHash\nFirst public report on\nProject Wood by\nFrankie Li\nFirst public\nreport on DCM\nby Jianming\n\nTencent publishes\na report on\nDCM/Dark Specter\n\nNSPX30 cluster\ntracked as\nBlackwood\n\nTargeted regions\n\nBlackwood profile\n\nJapan\n\nChina\n\nSectors\nUnited Kingdom\n\nManufacturing\n\nToolkit\n\nNSPX30\n\nAitM\n\nIndividuals\n\nGeographical distribution of NSPX30 victims\nUnited Kingdom\nan individual\n\nGeographical distribution of NSPX30 victims\nUnited Kingdom\nan individual\n\nChina\nindividuals\na manufacturing and\ntrading company\na Japanese engineering\nand manufacturing\ncorporation\n\nGeographical distribution of NSPX30 victims\nUnited Kingdom\nan individual\n\nJapan\nindividuals\n\nChina\nindividuals\na manufacturing and\ntrading company\na Japanese engineering\nand manufacturing\ncorporation\n\nHow Blackwood uses AitM?\n\nWhat we observed on victim machines\nTencent QQ\nsoftware\nhttp://dl_dir.qq.com/invc/qq/minibrowser.zip\n(183.134.93.142)\n\n1. updates from\n\nLegitimate server\n\nTencentdl.exe\n\nQQ.exe\n\nWhat we observed on victim machines\nTencent QQ\nsoftware\nhttp://dl_dir.qq.com/invc/qq/minibrowser.zip\n(183.134.93.142)\n\n1. updates from\n\nLegitimate server\n\n2. writes to disk\n\nTencentdl.exe\n\nQQ.exe\n\nminibrowser_shell.dll\n\nWhat we observed on victim machines\nTencent QQ\nsoftware\nhttp://dl_dir.qq.com/invc/qq/minibrowser.zip\n(183.134.93.142)\n\n1. updates from\n\nLegitimate server\n\n2. writes to disk\n\nTencentdl.exe\n\nminibrowser_shell.dll\n\n3. loads DLL\n\nQQ.exe\n\nSuccessfully hijacked software updates by\nBlackwood and others!\n\nTencent QQ\n\nSogou Pinyin\n\nWPS Office\n\nSuccessfully hij…\n\nMISSION FAILURE\n\nWhat we learned from the attacks\n\nInitial access\nNSPX30 dropper downloaded\nthrough HTTP by legitimate\nsoftware.\n\nWhat we learned from the attacks\n\nInitial access\n\nLegitimate servers\n\nNSPX30 dropper downloaded\nthrough HTTP by legitimate\nsoftware.\n\nIP address associated to domains\nwere from legitimate\ninfrastructure.\n\nWhat we learned from the attacks\n\nInitial access\n\nLegitimate servers\n\nFlexible dropper\n\nNSPX30 dropper downloaded\nthrough HTTP by legitimate\nsoftware.\n\nIP address associated to domains\nwere from legitimate\ninfrastructure.\n\nNSPX30 dropper can be DLL/EXE,\nif required: in ZIP archive.\n\nThe NSPX30 and its design\n\nNSPX30 main features\n\nReliance on AitM\nNSPX30 dropper component is\ndelivered via hijacked updates.\nCommunicates with its\ninfrastructure through AitM.\n\nNSPX30 main features\n\nmsnsp.dll\n\nReliance on AitM\n\nPersistence\n\nNSPX30 dropper component is\ndelivered via hijacked updates.\n\nLoader persisted as a Winsock\nNamespace Package (NSP) DLL.\n\nCommunicates with its\ninfrastructure through AitM.\n\nMalicious DLL is automatically\nloaded into processes that use\nWinsock.\n\nNSPX30 main features\n\nmsnsp.dll\n\nReliance on AitM\n\nPersistence\n\nHighly modular\n\nNSPX30 dropper component is\ndelivered via hijacked updates.\n\nLoader persisted as a Winsock\nNamespace Package (NSP) DLL.\n\nCommunicates with its\ninfrastructure through AitM.\n\nMalicious DLL is automatically\nloaded into processes that use\nWinsock.\n\nComposed of many components:\nloaders, orchestrator, backdoor,\nand three groups of plugins.\n\nOrchestrator main functionality\nOrchestrator\nstarts\nThread 1\n\nstarts\n\nThread 2\n\nOrchestrator main functionality\nOrchestrator\nstarts\n\nstarts\n\nThread 2\n\nThread 1\n\ndownloads\nbackdoor\n\ndrops and\nloads\n\nBaidu website\n\nBackdoor\n\nOrchestrator main functionality\nOrchestrator\nstarts\n\nstarts\n\nThread 2\n\nThread 1\n\ndownloads\nbackdoor\n\ndrops and\nloads\n\nloads\n\nperforms\n\nBaidu website\n\nBackdoor\n\nPlugins\n\nAllowlisting\n\nOrchestrator plugins\nBackdoor\n\nc001.dat\n\nc002.dat\nFound! ☺\n\nOrchestrator\n\ndownloads\n\nloads\n\nc003.dat\n\nc005.dat\n\nc006.dat\nNot found \n\nThe orchestrator does not load a plugin named c004.dat!\n\nc007.dat\n\nOrchestrator plugins\nPlugin c001.dat\n\nPlugin c002.dat\n\nOrchestrator plugins\nPlugin c001.dat\n\ndecrypts\nkey from\n\nRegistry.db\n\nPlugin c002.dat\n\ndecrypt DBs\n\nQQ databases\n\nOrchestrator plugins\nPlugin c001.dat\n\ndecrypts\nkey from\n\nPlugin c002.dat\n\nhooks functions\nin memory\n\ndecrypt DBs\n\nissues SQL\nstatements\nRegistry.db\n\nQQ databases\n\nCommon.dll\n(SQL functions)\n\nKernelUtil.dll\n(Msg functions)\n\nOrchestrator plugins\nInput stream APIs\n• waveInOpen\n• waveInAddBuffer\n• waveInClose\nPlugin c003.dat\nwinmm.dll\n\nOutput stream APIs\n• waveOutOpen\n• waveOutWrite\n• waveOutClose\nCoCreateInstance -> IMMDeviceEnumerator\n\nole32.db\n\n“AITM is more than our favourite technique, it’s a way of life!” NSPX30’s developers, maybe.\n\nAllowlisting in security software\n\nTencent PC\nManager\n\n360 Safeguard\nAnd Antivirus\n\nKingsoft Antivirus\n\nAllowlisting in Tencent PC Manager\nLoaders\n\nOrchestrator\n\nOr…\n\nmsnsp.dll\n\nUsed by NSPX30\n\nmshlp.dll\n\nWIN.cfg\n\nAllowlisting in 360 software\nTarget: 360 Antivirus\n\nsl2.db\n\nTarget: 360 Safeguard\n\nmshlp.dll\nmsnsp.dll\n\nUsed by: NSPX30 and Gelsemium.\n\nUsed by: NSPX30, Gelsemium, and McRAT.\n\nAllowlisting in Kingsoft Antivirus\n\nmsnsp.dll\n\nUsed by: NSPX30\n\nBonus mention: allowlisting in Windows Defender\n\nOverview of the backdoor’s functionality\na010.dat\nGather system information\nExtracts embedded plugins\nmsfmtkl.dat\n(Backdoor)\n\nLoad plugins from disk\nConfigure plugins\nDownload plugins\nCommunicate with C&C\nExfiltrate collected data\n\nAitM\n\nb010.dat\n\nb011.dat\n\nAnonymizing the attacker’s\ninfrastructure via AitM\n\nHow we believe Blackwood operates\n\nLocal network\n\nBlackwood\n\nRouter\n\nUnknown\nnetwork implant\n\nHow we believe Blackwood operates\nLocal network\nTraffic from\nunencrypted\nupdate protocol\n\nNSPX30\n\nTencent QQ\nserver\n\nRouter\nhttp://dl_dir.qq.com/invc/qq/minibrowser.zip\n(183.134.93.142)\n\nHow we believe Blackwood operates\nLocal network\nNSPX30\npackets\nappear\nlegitimate.\n\nNSPX30 packets are\nforwarded to the\nreal attacker’s infra\n\nDownloading components\n\nFingerprint\n\nhttp://www.baidu.com\n\n/id=<ID>&ad=32&os=<OS_MAJOR_VERSION>.<OS_MINOR_VERSION>&t=<ATTEMPTS>\n\nMozilla/4.0 (compatible;MSIE 5.0; Windows 98)\n\nUDP interception\nFingerprint\n\nDNS server at 180.76.76.11/24\nPort: 53, 4499, 8000\nTransaction ID always 0xFEAD\nDomain: microsoft.com\n\nAppended data to exfiltrate\n\nUDP interception\nBeginning at 180.76.76.11\n\nPort: 53 or 4499 or 8000\n\nBaidu DNS is at 180.76.76.76\n\nWhat about ISP infrastructure compromise?\n\nWhat about ISP compromise?\n\nGlobal reach\nNot all targets are located in\nChina.\n\nWhat about ISP compromise?\n\nGlobal reach\n\nNot always China\n\nNot all targets are located in\nChina.\n\nSome servers from Baidu\nnetwork 186.76.76.0/24 are\nanycast: they could be\ngeolocated around the world.\n\nWhat about ISP compromise?\n\nGlobal reach\n\nNot always China!\n\nReliability\n\nNot all targets are located in\nChina.\n\nSome servers from Baidu\nnetwork 186.76.76.0/24 are\nanycast: they could be\ngeolocated around the world.\n\nAitM mechanism appears to be\nreliable. Exfiltration requires A\nLOT of packets.\n\nAssessment: ISP compromise for AitM is unlikely.\n\nThe China-aligned AitM club\n\nChina-aligned APTs with AitM capability tracked by ESET\n\nEvasive Panda\n\nLittleBear\n\nAitM via compromised network device, or ISP? We don’t know.\n\nLuoYu\n\nBlackwood\n\nAitM working outside of China networks? Yes.\n\nThe update hijacking mechanism seems suspiciously similar for all four clusters\n\nTheWizards APT\n\nTargeted regions\nPhilippines\n\nHong Kong\n\nSectors\nUAE\n\nChina\n\nGambling\n\nToolkit\n\nWizardNet\n\nDarkNights\n\nSpellbinder\n\nIndividuals\n\nTheWizards APT\n\nTargeted regions\nPhilippines\n\nHong Kong\n\nSectors\nUAE\n\nChina\n\nGambling\n\nIndividuals\n\nToolkit\n\nWizardNet\n\nDarkNights\n\nSpellbinder\n\nSpellbinder: lateral\nmovement tool that\nperforms AitM via IPv6\nStateless Address\nAutoconfiguration\n(SLAAC) spoofing attack\n\nTheWizards approach to AitM\nICMPv6 RA message\nPreffix:\n2001:db8::/64\nRDNSS:\n240e:56:4000:8000::11\n240e:56:4000:8000::22\n\nSpellbinder\n\nVictim\n\nTheWizards approach to AitM\nICMPv6 RA message\nPreffix:\n2001:db8::/64\nRDNSS:\n240e:56:4000:8000::11\n240e:56:4000:8000::22\n\nSpellbinder\n\nVictim\n\nTheWizards approach to AitM\nICMPv6 RA message\nPreffix:\n2001:db8::/64\nRDNSS:\n240e:56:4000:8000::11\n240e:56:4000:8000::22\n\nDNS query\nget.sogou.com\n\nSpellbinder\n\nVictim\n\nTheWizards approach to AitM\nICMPv6 RA message\nPreffix:\n2001:db8::/64\nRDNSS:\n240e:56:4000:8000::11\n240e:56:4000:8000::22\n\nDNS query\nget.sogou.com\n\nSpellbinder\n\nDNS reply\nmalicious server IP\n\nAttacker’s server\n\nVictim\n\nTheWizards approach to AitM\nICMPv6 RA message\nPreffix:\n2001:db8::/64\nRDNSS:\n240e:56:4000:8000::11\n240e:56:4000:8000::22\n\nDNS query\nget.sogou.com\n\nSpellbinder\n\nDNS reply\nmalicious server IP\n\nVictim\nget update via\nHTTP\n\nAttacker’s server\n\nDropper DLL\n\nSpellbinder’s IPv6 SLAAC attack\nAttack vector discussed by the IETF as early as 2008\nIPv6 is enabled by default on modern Windows OS\nVery effective:\n▪ Dozens of machines compromised in a short time\n▪ No noticeable effect for the victims\n\nConclusion\n\nWhat is known and what is not known\n• NSPX30 and DCM for Win32\n• Project Wood for Win32\n\nProject Wood is alive and well:\n• Linux version recently found!\n\nWhat is known and what is not known\n• NSPX30 and DCM for Win32\n• Project Wood for Win32\n\nIs NSPX30 developed by a digital\nquartermaster?\n\nProject Wood is alive and well:\n• Linux version recently found!\n\nWhat is known and what is not known\n• NSPX30 and DCM for Win32\n• Project Wood for Win32\n\nProject Wood is alive and well:\n• Linux version recently found!\n\nIs NSPX30 developed by a digital\nquartermaster?\nHow is Blackwood able to accurately\nfind its victims in cyberspace?\n\nReports by CitizenLab and McAfee\n\nWhat is known and what is not known\n• NSPX30 and DCM for Win32\n• Project Wood for Win32\n\nProject Wood is alive and well:\n• Linux version recently found!\n\nIs NSPX30 developed by a digital\nquartermaster?\nHow is Blackwood able to accurately\nfind its victims in cyberspace?\n\nThe elusive AitM network implant\n\nどうも ありがとう ございます\nQ&A",
      "language": "EN",
      "sources": [
        {
          "id": "bf5be533-fa31-4590-ae37-5761c97ffa34",
          "created_at": "2022-10-25T16:13:58.389257Z",
          "updated_at": "2022-10-25T16:13:58.389257Z",
          "deleted_at": null,
          "name": "Malpedia",
          "url": "https://malpedia.caad.fkie.fraunhofer.de",
          "description": "Malpedia is a free service offered by Fraunhofer FKIE",
          "reports": null
        }
      ],
      "references": [
        "https://jsac.jpcert.or.jp/archive/2024/pdf/JSAC2024_1_2_facundo_en.pdf"
      ],
      "report_names": [
        "JSAC2024_1_2_facundo_en.pdf"
      ],
      "threat_actors": [
        {
          "id": "19ac84cc-bb2d-4e0c-ace0-5a7659d89ac7",
          "created_at": "2022-10-25T16:07:23.422755Z",
          "updated_at": "2024-02-06T02:02:35.326951Z",
          "deleted_at": null,
          "main_name": "Bronze Highland",
          "aliases": [
            "Daggerfly",
            "Evasive Panda"
          ],
          "source_name": "ETDA:Bronze Highland",
          "tools": [
            "Agentemis",
            "Cobalt Strike",
            "CobaltStrike",
            "KsRemote",
            "LOLBAS",
            "LOLBins",
            "Living off the Land",
            "MgBot",
            "Mgmbot",
            "cobeacon"
          ],
          "source_id": "ETDA",
          "reports": null
        },
        {
          "id": "b5550c4e-943a-45ea-bf67-875b989ee4c4",
          "created_at": "2022-10-25T16:07:23.675771Z",
          "updated_at": "2024-02-06T02:02:35.470887Z",
          "deleted_at": null,
          "main_name": "Gelsemium",
          "aliases": [
            "Operation NightScout",
            "Operation TooHash"
          ],
          "source_name": "ETDA:Gelsemium",
          "tools": [
            "ASPXSpy",
            "ASPXTool",
            "Agentemis",
            "BadPotato",
            "CHINACHOPPER",
            "China Chopper",
            "Chrommme",
            "Cobalt Strike",
            "CobaltStrike",
            "Gelsemine",
            "Gelsenicine",
            "Gelsevirine",
            "JuicyPotato",
            "OwlProxy",
            "Owowa",
            "SessionManager",
            "SinoChopper",
            "SpoolFool",
            "SweetPotato",
            "cobeacon",
            "reGeorg"
          ],
          "source_id": "ETDA",
          "reports": null
        },
        {
          "id": "77874718-7ad2-4d15-9831-10935ab9bcbe",
          "created_at": "2022-10-25T15:50:23.619911Z",
          "updated_at": "2024-02-06T02:00:51.389537Z",
          "deleted_at": null,
          "main_name": "Gelsemium",
          "aliases": [
            "Gelsemium"
          ],
          "source_name": "MITRE:Gelsemium",
          "tools": [
            "Gelsemium",
            "Mimikatz"
          ],
          "source_id": "MITRE",
          "reports": null
        },
        {
          "id": "2d4d2356-8f9e-464d-afc6-2403ce8cf424",
          "created_at": "2023-01-06T13:46:39.290101Z",
          "updated_at": "2024-02-06T02:00:03.494774Z",
          "deleted_at": null,
          "main_name": "Gelsemium",
          "aliases": [
            "狼毒草"
          ],
          "source_name": "MISPGALAXY:Gelsemium",
          "tools": [],
          "source_id": "MISPGALAXY",
          "reports": null
        },
        {
          "id": "98411ff0-e420-4eef-9943-e93d9f3c95b9",
          "created_at": "2022-10-25T16:47:55.580304Z",
          "updated_at": "2024-02-06T02:05:53.774143Z",
          "deleted_at": null,
          "main_name": "BRONZE HIGHLAND",
          "aliases": [
            "Evasive Panda "
          ],
          "source_name": "Secureworks:BRONZE HIGHLAND",
          "tools": [
            " KsRemote",
            " mgbot",
            "Cobalt Strike"
          ],
          "source_id": "Secureworks",
          "reports": null
        },
        {
          "id": "812f36f8-e82b-41b6-b9ec-0d23ab0ad6b7",
          "created_at": "2023-01-06T13:46:39.413725Z",
          "updated_at": "2024-02-06T02:00:03.580273Z",
          "deleted_at": null,
          "main_name": "BRONZE HIGHLAND",
          "aliases": [
            "Evasive Panda",
            "Daggerfly"
          ],
          "source_name": "MISPGALAXY:BRONZE HIGHLAND",
          "tools": [],
          "source_id": "MISPGALAXY",
          "reports": null
        },
        {
          "id": "b72c2616-cc7c-4c47-a83d-6b7866b94746",
          "created_at": "2023-01-06T13:46:39.425297Z",
          "updated_at": "2024-02-06T02:00:03.58824Z",
          "deleted_at": null,
          "main_name": "Red Nue",
          "aliases": [
            "LuoYu"
          ],
          "source_name": "MISPGALAXY:Red Nue",
          "tools": [],
          "source_id": "MISPGALAXY",
          "reports": null
        },
        {
          "id": "f35997d9-ca1e-453f-b968-0e675cc16d97",
          "created_at": "2023-01-06T13:46:39.490819Z",
          "updated_at": "2024-02-06T02:00:03.627976Z",
          "deleted_at": null,
          "main_name": "Evasive Panda",
          "aliases": [
            "BRONZE HIGHLAND"
          ],
          "source_name": "MISPGALAXY:Evasive Panda",
          "tools": [],
          "source_id": "MISPGALAXY",
          "reports": null
        },
        {
          "id": "c13153a4-8dda-4cc5-ac31-c9ca25f3563c",
          "created_at": "2024-02-01T02:00:04.227755Z",
          "updated_at": "2024-02-06T02:00:04.018039Z",
          "deleted_at": null,
          "main_name": "Blackwood",
          "aliases": [],
          "source_name": "MISPGALAXY:Blackwood",
          "tools": [],
          "source_id": "MISPGALAXY",
          "reports": null
        }
      ],
      "ts_created_at": 1706926020,
      "ts_updated_at": 1707185893,
      "ts_creation_date": 1705924303,
      "ts_modification_date": 1705924303,
      "files": {
        "pdf": "https://pub-7cb8ac806c1b4c4383e585c474a24719.r2.dev/afa62ace20b3489a9b137a3b8091813c9d96373d.pdf",
        "text": "https://pub-7cb8ac806c1b4c4383e585c474a24719.r2.dev/afa62ace20b3489a9b137a3b8091813c9d96373d.txt",
        "img": "https://pub-7cb8ac806c1b4c4383e585c474a24719.r2.dev/afa62ace20b3489a9b137a3b8091813c9d96373d.jpg"
      }
    }
  ],
  "message": "library entries",
  "status": "success"
}

**Comment by sameershaik-coder** [comment]
I have started working on this connector development

**Comment by sameershaik-coder** [comment]
I have completed development of connector, will start testing and update on once the testing is complete

**Comment by sameershaik-coder** [comment]
sorry everyone, i have to pause this connector development due to other commitments. I keep getting RabbitMQ error when i run the containers but no issues when i run the code directly. I dont have enough time to debug this and fix it. The moment i get some time to resume this, i will update here. until then consider this as on hold from my end.
